search_service/models/dashboard.py

```diff
@@ -40,3 +42,15 @@ class DashboardSchema(DashboardSummarySchema):
     class Meta:
         target = Dashboard
         register_as_scheme = True
+
+
+@attr.s(auto_attribs=True, kw_only=True)
```

Comment:
What's the reason this can't use the `DashboardSummary`?

Reply:
This actually encapsulate `DashboardSummary`.
https://github.com/lyft/amundsensearchlibrary/pull/108/files#diff-7a3f4e6680512c4f1bfa23c826dfd639R13

You can think this is `list` of `DashboardSummary` with `total_result`

Reply:
Oh I see it now. Thanks for the explanation.